### DISCUSSION TITLE
FLUX Tools 21 nov 2024 - ControlNet model

### DISCUSSION THREAD

**Discussion by SunGreen777** [lllyasviel/stable-diffusion-webui-forge#2364] [General]
FLUX Tools. 
https://blackforestlabs.ai/
https://huggingface.co/black-forest-labs
ControlNet model
Will you be able to screw it on, what are the hardware requirements?

**Comment by JackDainzh** [comment]
It's not a controlnet model, it's an entirely separate model that is designed to be used as a model to generate, not to guide. The issue is that, as of now, currently with img2img implementation of flux, there is no way to guide the model, say, with pix2pix intrsuct's Image CFG Scale slider, because it doesn't effect anything at the moment. (I forced it visible in the ui, when using flux), but because flux's conditioning is different to that of regular sd models, it get's skipped.
To implement the guidence from img2img tab, is to rewrite the whole backend engine, which I have no idea how long will it take, maybe months, or maybe 1 day.

**Comment by SunGreen777** [comment]
Did I understand correctly? "Flux tools" will not be in Forge?

**Reply by JackDainzh** [reply]
I didn't say that.
